src/OidcClientSettings.ts

```diff
@@ -157,10 +165,12 @@ export class OidcClientSettingsStore {
     public readonly client_id: string;
     public readonly client_secret: string | undefined;
     public readonly response_type: string;
-    public readonly scope: string;
+    public readonly scope: string | undefined;
     public readonly redirect_uri: string;
     public readonly post_logout_redirect_uri: string | undefined;
-    public readonly client_authentication: "client_secret_basic" | "client_secret_post";
+    public readonly client_authentication:
```

Comment:
I have seen multiple unnecessary and not related code style changes, please remove them...

Reply:
doh. forgot to turn off the auto-formatter on save. These should be fixed now.